src/Microsoft.OData.ModelBuilder/Vocabularies/Capabilities/Constants.cs

```diff
@@ -0,0 +1,212 @@
+
+            /// <summary>Property Description of Org.OData.Capabilities.V1.DeleteRestrictions.</summary>
+            public const string Description = "Description";
+
+            /// <summary>Property LongDescription of Org.OData.Capabilities.V1.DeleteRestrictions.</summary>
+            public const string LongDescription = "LongDescription";
+        }
+
+        internal static class ReadRestrictions
+        {
+            /// <summary>Org.OData.Capabilities.V1.ReadRestrictions</summary>
+            public const string Term = "Org.OData.Capabilities.V1.ReadRestrictions";
+
+            /// <summary>Property ReadByKeyRestrictions of Org.OData.Capabilities.V1.ReadRestrictions.</summary>
+            public const string ReadByKeyRestrictions = "ReadByKeyRestrictions";
+        }
+
+        internal static class ReadRestrictionsBase
+        {
+            /// <summary>Property Readable of Org.OData.Capabilities.V1.ReadRestrictionsBase</summary>
+            public const string Readable = "Readable";
+
+            /// <summary>Property Permissions of Org.OData.Capabilities.V1.ReadRestrictionsBase.</summary>
+            public const string Permissions = "Permissions";
+
+            /// <summary>Property Description of Org.OData.Capabilities.V1.ReadRestrictionsBase.</summary>
+            public const string Description = "Description";
+
+            /// <summary>Property LongDescription of Org.OData.Capabilities.V1.ReadRestrictionsBase.</summary>
+            public const string LongDescription = "LongDescription";
+        }
+
+        internal static class OperationRestrictions
+        {
+            /// <summary>Org.OData.Capabilities.V1.OperationRestrictions</summary>
+            public const string Term = "Org.OData.Capabilities.V1.OperationRestrictions";
+
+            /// <summary>Property FilterSegmentSupported of Org.OData.Capabilities.V1.InsertRestrictions.</summary>
+            public const string FilterSegmentSupported = "FilterSegmentSupported";
+
+            /// <summary>Property Permissions of Org.OData.Capabilities.V1.InsertRestrictions.</summary>
+            public const string Permissions = "Permissions";
+        }
+
+        internal static class PermissionType
+        {
+            /// <summary>Property SchemeName of Org.OData.Capabilities.V1.PermissionType.</summary>
+            public const string SchemeName = "SchemeName";
+
+            /// <summary>Property Scopes of Org.OData.Capabilities.V1.PermissionType.</summary>
+            public const string Scopes = "Scopes";
+        }
+
+        internal static class ScopeType
+        {
+            /// <summary>Property Scope of Org.OData.Capabilities.V1.ScopeType.</summary>
+            public const string Scope = "Scope";
+
+            /// <summary>Property RestrictedProperties of Org.OData.Capabilities.V1.ScopeType.</summary>
+            public const string RestrictedProperties = "RestrictedProperties";
+        }
+    }
+}
```

Comment:
General questions in regard to the constants you chose to represent here... Was there any specific criteria you followed in determining which ones to represent/include or not to represent? Just as an example, why would `NonInsertableProperties` and `NotInsertableNavigationProperties` not make the cut in `InsertRestrictions` constants?